vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt

```diff
@@ -468,7 +468,6 @@ class NotificationUtils @Inject constructor(private val context: Context,
                         setSmallIcon(R.drawable.ic_call_answer)
                     }
                 }
-                // This is a trick to make the previous notification with same id disappear as cancel notification is not working with Foreground Service.
```

Comment:
So also remove the trick itself?

Reply:
Nope, it's not really a trick anymore, cancelNotification work, but only when stopForeground is called.

Reply:
OK, thanks